### DISCUSSION TITLE
Where's the e-mail-invitation gone?

### DISCUSSION THREAD

**Discussion by n1smithy** [Ylianst/MeshCentral#6607] [Q&A] [answered]
Hello @ all

I feel dumb. I cannot find the feature to send invitations to download and install the agent by e-mail.

Can this be disabled by any wrong or missing settings?

Thanks for all help!

**Comment by si458** [comment]
It's only visible when you have the smtp setup in config.json

**Comment by n1smithy** [comment]
But I have it?! Hm …

**Reply by si458** [reply]
Well can you share your config.json then?
Also when you start meshcentral does the logs show smtp enabled in the output?

**Reply by n1smithy** [reply]
I do not find the logs. But as I started the docker-container not as a daemon, there was a strange message about "Server smtp has no users (…)" or stuff. Strange. I'll post the config.json as soon as I removed all sensitive data.

**Reply by n1smithy** [reply, marked as answer]
OK, finally found my mistake by comparing a sample-config.json with my one. The smtp-section was not correctly set in my config.json. I had corrupted it several weeks ago and had to correct it. Seems that my correction led to a move of the smtp-section into the wrong hierachicle place.

However: E-Mail-invitation is now available again – phew!

**Reply by n1smithy** [reply]
By the way: @si458 thank you, Simon, for your imediately response!
